### DISCUSSION TITLE
GitHub benefits as a student's "Continue" button is not working

### DISCUSSION THREAD

**Discussion by gheroufosse** [community/community#60581] [GitHub Education] [answered]
### Hi everyone,

I would like to get my GitHub benefits as a student, but the "Continue" button is actually not working, despite my University being recognised.
![image](https://github.com/community/community/assets/88828575/c2ee2413-142d-4e3c-92fc-f7e444cfb703)

The button stays grey once it has been clicked on, and nothing is moving/charging.

Does anyone has a idea how to bypass this problem ? 
Thanks

**Comment by OshekharO** [comment, marked as answer]
I have seen this same issue happening with my friend, browser failed to get his location. He switched to MS edge and tried again and it worked. Or check in developer console for log what error it gives after button being clicked.

Hopefully this answer satisfy you.

**Reply by gheroufosse** [reply]
Thanks, it works indeed using Safari instead of Brave

**Reply by Bristiraj** [reply]
still doesn't work for me

**Reply by GuiSousa2910** [reply]
It's okay

**Reply by Mo99411** [reply]
It's okay

**Reply by ghost** [reply]
thanks

**Reply by fiasco-zh** [reply]
thanks

**Reply by salutonl** [reply]
thanks, it works

**Reply by Joel15797** [reply]
Unfortunately, it doesn't work for me even when using Edge.

**Comment by jemshit** [comment]
You might have text like this above emails:

> You were last verified as a student on [Month]. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.

I right clicked disabled submit button, removed "disabled" attribute and clicked. Then it showed _"Hi, [Name]! You were last verified as a student on . It is not necessary for you to re verify at this time."_

Github education UX is shit

**Reply by VishruthVS** [reply]
What next?

**Reply by dotLuxPI** [reply]
This is still not working form me, I tried multiple browsers, PCs and even tried to remove the 'disabled' attribute from the button and clicking it again, but I'm still blocked at the location check

**Reply by Jesus-QC** [reply]
> This is still not working form me, I tried multiple browsers, PCs and even tried to remove the 'disabled' attribute from the button and clicking it again, but I'm still blocked at the location check

did it end up working for you?

**Comment by Mo99411** [comment]
Ok

**Comment by Mo99411** [comment]
Hopefully this answer satisfy you

**Comment by masabia** [comment]
Hi I reopen this issue because changing browser is not working for me.

**Comment by MrTimonM** [comment]
For me too, I can't click continue 
I have to reverify my studentship

**Comment by eddie93ie** [comment]
I cant click continue as well, saying i need to verify school email. It is already verified

**Comment by MrAnimal7** [comment]
It looks like the **domain name** part of the email has to be **exactly** as in the list of emails provided and is **case sensitive**. At first I copied and pasted my student email from my profile in Teams which had some uppercase letters in it. Only after re-registering the email using all **lower-case** letters I was able to proceed with the application. That's what worked for me anyway.

**Comment by roscosmo** [comment]
still not fixed? tried 3 browsers
